### ISSUE TITLE
ParameterNumberCheck should not check overriden methods

### ISSUE THREAD

**Issue by golszewski86** [yegor256/qulice#1062]
Consider the following code:
```java
public interface ExternalInterface {
    void foo(String first, String second, String third, String fourth);
}

public final class MyImpl implements ExternalInterface {
    @Override
    public void foo(final String first, final String second, final String third, final String fourth) {
        // do nothing
    }
}
```

For MyImpl class the following warning is reported:

> More than 3 parameters (found 4). (ParameterNumberCheck)

However, the warning should not be triggered. The `foo` method is inherited hence overriden `foo` method in `MyImpl` must accept 4 parameters.
This is especially true when the external interface comes from 3rd party library and cannot be changed.

I believe ParameterNumberCheck should not be triggered for overriden methods, but only for method declarations.

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) please, pay attention to this issue

**Comment by 0crat** [comment]
@golszewski86[/z](https://www.0crat.com/u/golszewski86) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3T49A35L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@golszewski86 I believe it's fixed, try version 0.20.0
